### DISCUSSION TITLE
Update to latest prisma version

### DISCUSSION THREAD

**Discussion by Manubi** [prisma/nexus-prisma#103] [General]
Hey,
Is it a lot of work to keep the plugin compatible with the latest prisma version?
thanks!

**Comment by dominichadfield-jelly** [comment]
I think there is greater impetus to update when there are breaking changes or a large extension to the feature set. The past few Prisma releases do not have very large changes so you can safely ignore the warnings in the terminal.

**Reply by Manubi** [reply]
Thanks for the info. I just wanted to mention that It would help if it would be tested against the latest version and just get a minor official pump with the notification that it's working with the latest prisma version.
I know it works right now with the latest version, but I don't like to ignore errors.
